adam-core/src/main/scala/org/bdgenomics/adam/converters/AlignmentRecordConverter.scala

```diff
@@ -30,31 +30,25 @@ import scala.collection.JavaConversions._
 class AlignmentRecordConverter extends Serializable {
 
   /**
-   * Converts a single record to FASTQ. FASTQ format is:
-   *
-   * {{{
-   * @readName
-   * sequence
-   * +<optional readname>
-   * ASCII quality scores
-   * }}}
+   * Prepare a single record for conversion to FASTQ and similar formats.
```

Comment:
I would expand what you mean by "prepare" here. Once I read through the method, I understood what you meant, but ideally I shouldn't need to read the source first.